tests/VirtoCommerce.ElasticSearchModule.Tests/ElasticSearchRequestBuilderTests.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using AutoFixture;
+using FluentAssertions;
+using Nest;
+using VirtoCommerce.ElasticSearchModule.Data;
+using VirtoCommerce.SearchModule.Core.Model;
+using Xunit;
+
+namespace VirtoCommerce.ElasticSearchModule.Tests
+{
+    [Trait("Category", "CI")]
+    public class ElasticSearchRequestBuilderTests
+    {
+        private readonly ElasticSearchRequestBuilderTestProxy _testClass = new ElasticSearchRequestBuilderTestProxy();
+        private readonly Fixture _fixture = new Fixture();
+
+        [Theory]
+        [InlineData("0", "false")]
+        [InlineData("1", "true")]
+        [InlineData("true", "true")]
+        [InlineData("false", "false")]
+        [InlineData("tRuE", "true")]
+        [InlineData("FaLsE", "false")]
+        public void CreateTermFilter_BooleanAggregate_ShouldCreateCorrectValues(string value, string convertedValue)
+        {
+            // Arrange
+            var fieldName = _fixture.Create<string>();
```

Comment:
Do we need fixture for one string?

Reply:
I prefer working with randomized data when this data does not participate in business logic. In these tests, we do not care about data in fieldName, but we need it in two cases.